### PR TITLE
Mals 1

### PR DESCRIPTION
**Pull request by malvikaRavichandran**

### CODE REVIEW

**Review by amoghkokari — COMMENTED** [review]
please attach screenshots

**Review by amoghkokari — CHANGES_REQUESTED** [review]
update the images in readme please

**Comment by malvikaRavichandran** [comment]
> please attach screenshots



> update the images in readme please

added

**Comment by malvikaRavichandran** [comment]
> update the images in readme please

added

File: main.py [hunk 1]
```diff
@@ -19,12 +19,14 @@ def student():
 
 @app.route('/result',methods = ['POST', 'GET'])
 def result():
+   global channel_name
```
Comment (amoghkokari) [inline main.py]: use cred.py as future migration to db will be easier

**Review by amoghkokari — CHANGES_REQUESTED** [review]
check comments

File: main.py (hunk 1 above)
Comment (malvikaRavichandran) [reply main.py]: done

File: main.py [hunk 2]
```diff
@@ -25,6 +25,8 @@ def result():
 
       result = request.form
       df, channel_id = process_result(result["ch1"])
+      channel_name=result.to_dict().get('ch1').split("/")[-1]
```
Comment (amoghkokari) [inline main.py]: redundant, consider directly storing to cred.cha....

File: templates/predict.html [hunk 3]
```diff
@@ -1,49 +1,77 @@
...
-        {% endfor %}
-        </table>
-        {% endif %}
-    </div> 
-     
-   </body>
+
+<body>
+  <div style="text-align: center">
+    <h1>Enter YouTube Video Title and Description for channel - {{channel_name}}</h1>
```
Comment (amoghkokari) [inline templates/predict.html]: use "channel_name" inlace of -

File: templates/result.html [hunk 4]
```diff
@@ -1,6 +1,6 @@
 <html>
     <body>
-        <h1>Word Clouds on all the video data present in the Channel</h1>
```
Comment (amoghkokari) [inline templates/result.html:3]: ^^^^

**Review by amoghkokari — CHANGES_REQUESTED** [review]
address and resolve the comments

File: templates/result.html (hunk 4 above)
Comment (malvikaRavichandran) [reply templates/result.html:3]: done

**Review by amoghkokari — APPROVED** [review]
good work

**Pull request merged**